Formula/alot.rb

```diff
@@ -0,0 +1,87 @@
+class Alot < Formula
+  desc "Terminal-based mail user agent for the notmuch mail system"
+  homepage "https://github.com/pazz/alot"
+  url "https://github.com/pazz/alot/archive/0.3.7.tar.gz"
+  sha256 "fd2d4121ba109224560919e84e320b09526891fbc0038afcea26a3f1284dad26"
+
+  option "without-docs", "Don't build documentation"
+
+  depends_on "gpgme"
+  depends_on "libmagic"
+  depends_on "notmuch" => "with-python"
+  depends_on "sphinx-doc" => :build if build.with? "docs"
+
+  resource "six" do
+    url "https://pypi.python.org/packages/source/s/six/six-1.10.0.tar.gz"
+    sha256 "105f8d68616f8248e24bf0e9372ef04d3cc10104f1980f54d57b2ce73a5ad56a"
+  end
+
+  resource "configobj" do
+    url "https://pypi.python.org/packages/source/c/configobj/configobj-5.0.6.tar.gz"
+    sha256 "a2f5650770e1c87fb335af19a9b7eb73fc05ccf22144eb68db7d00cd2bcb0902"
+  end
+
+  resource "zope.interface" do
+    url "https://pypi.python.org/packages/source/z/zope.interface/zope.interface-4.1.3.tar.gz"
+    sha256 "2e221a9eec7ccc58889a278ea13dcfed5ef939d80b07819a9a8b3cb1c681484f"
+  end
+
+  resource "twisted" do
+    url "https://pypi.python.org/packages/18/85/eb7af503356e933061bf1220033c3a85bad0dbc5035dfd9a97f1e900dfcb/Twisted-16.2.0.tar.bz2"
+    sha256 "a090e8dc675e97fb20c3bb5f8114ae94169f4e29fd3b3cbede35705fd3cdbd79"
+  end
+
+  resource "urwid" do
+    url "https://pypi.python.org/packages/source/u/urwid/urwid-1.3.1.tar.gz"
+    sha256 "cfcec03e36de25a1073e2e35c2c7b0cc6969b85745715c3a025a31d9786896a1"
+  end
+
+  resource "urwidtrees" do
+    url "https://github.com/pazz/urwidtrees/archive/1.0.2.tar.gz"
+    sha256 "703f4b161b930a26a461a3e3e695f94237ac75e2a52b87613e49093d9aa76034"
+  end
+
+  resource "pygpgme" do
+    url "https://pypi.python.org/packages/source/p/pygpgme/pygpgme-0.3.tar.gz"
+    sha256 "5fd887c407015296a8fd3f4b867fe0fcca3179de97ccde90449853a3dfb802e1"
+  end
+
+  resource "python-magic" do
+    url "https://pypi.python.org/packages/source/p/python-magic/python-magic-0.4.11.tar.gz"
+    sha256 "89021e288d6efd22cde2842349d79939b9664efdbf99f5790c9862a67759ea94"
+  end
+
+  def install
+    ENV["PYTHONPATH"] = libexec/"vendor/lib/python2.7/site-packages"
+    resources.each do |r|
+      r.stage do
+        system "python", *Language::Python.setup_install_args(libexec/"vendor")
+      end
+    end
+    touch libexec/"vendor/lib/python2.7/site-packages/zope/__init__.py"
+
+    args = Language::Python.setup_install_args(prefix)
+    args << "--install-scripts=#{libexec}"
+    system "python", *args
+    (bin/"alot").write_env_script("#{libexec}/alot", :PYTHONPATH => ENV["PYTHONPATH"])
+
+    pkgshare.install Dir["extra/*"] - %w[extra/completion]
+    zsh_completion.install "extra/completion/alot-completion.zsh" => "_alot"
+
+    if build.with? "docs"
+      sphinx = Formula["sphinx-doc"]
+      ENV.prepend_path "PYTHONPATH", sphinx.libexec/"vendor/lib/python2.7/site-packages"
```

Comment:
I don't quite understand what you're doing in this block. Why are you using the `sphinx-build` from `libexec` rather than the one from `bin` which has built-in env scripting and doesn't need other formulae to fiddle with the `PYTHONPATH` to use it?